### ISSUE TITLE
Missing webm and mp4 in nginx default conf

### ISSUE THREAD

**Issue by nowheretobefound** [hoellen/docker-nextcloud#93]
FirstRunWizard videos doesnt work. After the first login to a new user account I see a blue page but no video. 

I think Im running into this issue :

https://github.com/nextcloud/firstrunwizard/issues/273

Nginx: Add mp4 and webm like below

location ~ \.(?:png|html|ttf|ico|jpg|jpeg|mp4|webm)$ {
    try_files $uri /index.php$request_uri;
}

**Comment by waja** [comment]
https://github.com/nextcloud/documentation/pull/1833/files is the relevant change of the config

**Comment by waja** [comment]
Yes. You can try to create your own config and bind mount it into the container by overriding the shipped one.
I've looked into the actual config documentation. Lookes like the recommended config was a bit restructured since the `default.conf` was created.

**Comment by waja** [comment]
> thank you, I guess this issue can be closed then :)
> 

I think you has the permission to close the issue as author.
